kotlin/samples/tictactoe/app/src/main/java/com/squareup/sample/mainactivity/MainActivity.kt

```diff
@@ -48,7 +49,10 @@ class MainActivity : AppCompatActivity() {
       finish()
     }
 
-    loggingSub = workflowRunner.renderings.subscribe { Timber.d("rendering: %s", it) }
+    loggingSub = CompositeDisposable(
+        workflowRunner.renderings.subscribe { Timber.d("rendering: %s", it) },
+        workflowRunner.debugSnapshots.subscribe { Timber.d("debug snapshot: %s", it) }
```

Comment:
Nit: `Timber.v` for the debug snapshot

Reply:
Fine, but why?

Reply:
Because I actually use those rendering lines when debugging, and they'll be harder to read if I can't easily filter out the debug snapshots.

Reply:
You can: just filter on `/rendering: /`.

Reply:
Then I'll only see the renderings. Having the full debug output seems verbose to me, so `.v` is my knee-jerk.

Reply:
Oh, I thought `d` was lower priority than `v` – it's the other way around. Makes sense now.